build_image.Dockerfile

```diff
@@ -45,22 +45,23 @@ RUN python3 -m pip install virtualenv==16.0.0
 RUN ln -s /usr/bin/python3.6 /usr/bin/python
 RUN ln -s /usr/bin/pip3 /usr/bin/pip
 
-RUN useradd -d /home/hca_cicd -ms /bin/bash -g root -G sudo hca_cicd
+ENV USER_HOME ${USER_HOME}
```

Comment:
Is ${USER_HOME} set by default on the base image we use?

Reply:
To add on to this, it's not just bash - any POSIX-compliant system will have `$HOME` set by default.